contrib/kind.sh

```diff
@@ -281,6 +282,23 @@ set_default_params() {
   fi
   OVN_HOST_NETWORK_NAMESPACE=${OVN_HOST_NETWORK_NAMESPACE:-ovn-host-network}
 
+  USING_PODMAN=0
+
+  if docker help | grep -q podman; then
+    echo "docker is aliased to podman"
```

Comment:
does this work for you?
I think that it always uses docker, if you want to use podman you have to set the environment variable
```
sudo KIND_EXPERIMENTAL_PROVIDER=podman kind create cluster
```

Reply:
I have the fedora package installed that aliases docker to podman

Reply:
(and by installing this package, kind uses podman without needing any overrides)

Reply:
but if you only have podman and you don't have docker installed, kind will use podman

Reply:
right but the script still requires that `docker` be meaningful.

Reply:
does it? I think that will be a bug, my memory is not good, but I think that we supported the case where docker is not present and only podman is

Reply:
It uses `docker build` as well as `docker exec`, so it definitely needs `docker` to work.

Reply:
Huh. Works fine on my machine :-)

Reply:
are you really using podman with kind?

Reply:
i think we should just use KIND_EXPERIMENTAL_PROVIDER=podman and not alias or use variables to mask the kind/docker executable with sudo. Force the user to run as root.

Reply:
basically, running `make` as root kind of stinks, which is why I initially wrote this.

Reply:
rootless support is very new in kind, and works both in docker and podman, actually docker was first in supporting it 
https://kind.sigs.k8s.io/docs/user/rootless/

However, rootless multinode wasn't working it uses this SLIRP thing to emulate the network